### PR TITLE
Setup starter footer

### PR DESCRIPTION
**Pull request by kadenzipfel**

### CODE REVIEW

File: components/footer/footer.sass [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  color: #f2f2f2 // Change to $light-neutral-color once color palette is merged
+
+.footer__info
+  display: flex
+  flex-direction: column
+  align-items: flex-start
+  justify-content: space-evenly
+  height: 100%
+  width: 33%
+  color: white
```
Comment (angusm) [inline components/footer/footer.sass]: Use a purpose-based color.

File: components/footer/footer.sass [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+.footer__link
+  color: #f2f2f2 // Change to $light-neutral-color once color palette is merged
+  transition: color .3s
+
+.footer__link:hover
+  color: white
+
+.footer__item
+  color: #f2f2f2 // Change to $light-neutral-color once color palette is merged
```
Comment (angusm) [inline components/footer/footer.sass]: Use a purpose-based color.

File: components/footer/footer.sass [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  height: 100%
+  width: 33%
+  color: white
+
+.footer__link
+  color: #f2f2f2 // Change to $light-neutral-color once color palette is merged
+  transition: color .3s
+
+.footer__link:hover
+  color: white
```
Comment (angusm) [inline components/footer/footer.sass]: Use a purpose-based color.

File: components/footer/footer.sass [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  display: flex
+  flex-direction: column
+  align-items: flex-start
+  justify-content: space-evenly
+  height: 100%
+  width: 33%
+  color: white
+
+.footer__link
+  color: #f2f2f2 // Change to $light-neutral-color once color palette is merged
```
Comment (angusm) [inline components/footer/footer.sass]: Use a purpose-based color.

File: components/footer/footer.sass [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  height: 100%
+
+.footer__links
+  display: flex
+  flex-direction: column
+  align-items: flex-start
+  justify-content: space-evenly
+  height: 100%
+  width: 33%
+  color: white
```
Comment (angusm) [inline components/footer/footer.sass]: Use a purpose-based color.

File: components/footer/footer.sass [hunk 6]
```diff
@@ -0,0 +1,39 @@
+.footer 
+  background: black // Change this to $dark-neutral-color once color palette is merged
```
Comment (angusm) [inline components/footer/footer.sass]: Use a purpose-based color.

File: components/footer/footer.html [hunk 7]
```diff
@@ -0,0 +1,33 @@
+<footer class="footer">
+  <img
+    src="{{ component.logo.url }}"
+    alt="{{ component.logo.alt }}"
+    class="footer__logo"
+  />
+  <ul class="footer__info">
+    <li class="footer__item">
+      Whelpton Electric
```
Comment (angusm) [inline components/footer/footer.html]: Move copy to a yaml file.

File: components/footer/footer.html [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+      </li>
+    </a>
+    <a href="tel:{{ component.phone }}" class="footer__link">
+      <li class="footer__item">
+        {{ component.phone }}
+      </li>
+    </a>
+  </ul>
+  <ul class="footer__links">
+    <a href="#" class="footer__link">
```
Comment (angusm) [inline components/footer/footer.html]: Pull in links from yaml and place in for loop.

**Review by angusm — CHANGES_REQUESTED** [review]

File: components/footer/footer.sass [hunk 9]
```diff
@@ -17,17 +17,17 @@
   justify-content: space-evenly
   height: 100%
   width: 33%
-  color: white
+  color: $background-white
```
Comment (angusm) [inline components/footer/footer.sass]: $background-white is not a purpose

**Review by angusm — CHANGES_REQUESTED** [review]

File: components/footer/footer.sass [hunk 10]
```diff
@@ -17,14 +17,14 @@
   justify-content: space-evenly
   height: 100%
   width: 33%
-  color: $background-white
+  color: $footer-text-white
```
Comment (angusm) [inline components/footer/footer.sass]: Change to $color-footer-text

**Review by angusm — CHANGES_REQUESTED** [review]

File: components/footer/footer.sass [hunk 11]
```diff
@@ -0,0 +1,38 @@
+.footer 
+  background: $light-neutral-color
```
Comment (angusm) [inline components/footer/footer.sass]: Let's use purpose based colors in this file.

File: components/footer/footer.sass [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+  margin: 0.5rem
+  font-weight: 300
+
+.footer__info
+  display: flex
+  flex-direction: column
+  align-items: center
+  justify-content: center
+  height: 100%
+  width: 33%
```
Comment (angusm) [inline components/footer/footer.sass]: How's this handle on very small screens? Should 33% be a max-width with a wider base width?
Just a question, if it handles good on small screens leave as is :)

File: content/pages/home.yaml [hunk 13]
```diff
@@ -40,3 +40,18 @@ components:
   - component: projects
     title: Projects
     id: projects
+  - component: footer
+    title: Footer
+    brand: !g.string home.company_name.text
+    logo:
+      url: https://i.imgur.com/qrovqR0.png
```
Comment (angusm) [inline content/pages/home.yaml]: Let's reference our images from the /static folder.
We never want to rely on a third-party hosting with a few exceptions (Google CDN for 3rd party libraries so we can leverage caching for instance)

**Review by angusm — CHANGES_REQUESTED** [review]

File: components/footer/footer.sass (hunk 12 above)
Comment (kadenzipfel) [reply components/footer/footer.sass]: Actually, just realized they're completely redundant with the new changes, thanks for pointing out

File: content/pages/home.yaml (hunk 13 above)
Comment (kadenzipfel) [reply content/pages/home.yaml]: Okay great, I was thinking that would be better but was having trouble figuring out where to place the images

**Review by angusm — CHANGES_REQUESTED** [review]
Just needs conflicts resolved and then all good

**Comment by angusm** [comment]
Looks like there's some new conflicts since merging in Feature/nav

**Pull request merged**